dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/ExtensionAliases.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.logging.data;
+
+import java.util.ArrayList;
+
+/**
+ *
+ */
+public class ExtensionAliases {
+    public ArrayList<String> originalExtensions;
+    public ArrayList<String> aliasesExtensions;
+
+    public ExtensionAliases() {
+        this.originalExtensions = new ArrayList<String>();
+        this.aliasesExtensions = new ArrayList<String>();
+    }
+
+    public void addExtensionAlias(String originalName, String alias) {
+        originalExtensions.add(originalName);
+        aliasesExtensions.add(alias);
+    }
+}
```

Comment:
this should probably be a static inner class inside NameAliases since NameAliases is the only class that uses it.